verifier.sh

```diff
@@ -348,18 +349,31 @@ _devtest_innervm_run () {
     ssh -t  $lxc_ip "rm -f ssh.log"
 
     ssh -t  $lxc_ip "sudo apt-get update"
+    # use this parameter to avoid blocks with sudoers updates: '-o Dpkg::Options::=--force-confdef'
     ssh -t  $lxc_ip "sudo apt-get -y upgrade"
+    ssh -t  $lxc_ip "wget http://ftp.openquake.org/mirror/mozilla/geckodriver-latest-linux64.tar.gz ; tar zxvf geckodriver-latest-linux64.tar.gz ; sudo cp geckodriver /usr/local/bin"
+    ssh -t  $lxc_ip "sudo pip install -U selenium==3.0.1"
```

Comment:
Be careful using `pip install -U`: it could update deps that you do not want to upgrade. I would skip `-U` if selenium is not present in the base lxc.